docs/glossary.md

```diff
@@ -0,0 +1,100 @@
+# InSpec Glossary
+
+## Basic Syntax
+```
+describe foo('/path/to/foo.txt') do
+    its('blah') { should cmp '123' }
+    it { should exist }
+    it { should be_reasonable }
+    it { should_not be_ridiculous }
+end
+```
+## Basic Elements:
+
+### describe **foo**, where
+
+  * `foo` is the _resource_
+
+### describe foo **('/path/to/foo.txt')**, where
+
+  * `('/path/to/foo.txt')` is the _resource parameter_
+
+## Tests:
+
+### **its('blah') { should cmp '123' }** is an _individual test_, where
+
+  * `'blah'` is a _property_
```

Comment:
Technically, the property is `blah` - the single quotes are not part of the property.

Reply:
OK